Sources/VLCActionSheet.swift

```diff
@@ -0,0 +1,422 @@
+/*****************************************************************************
+ * VLCActionSheet.swift
+ *
+ * Copyright © 2018 VLC authors and VideoLAN
+ * Copyright © 2018 Videolabs
+ *
+ * Authors: Soomin Lee <bubu@mikan.io>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+import UIKit
+
+// MARK: VLCRendererCollectionViewLayout
+class VLCRendererCollectionViewLayout: UICollectionViewFlowLayout {
+    override init() {
+        super.init()
+        setupLayout()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupLayout()
+    }
+
+    fileprivate func setupLayout() {
+        minimumLineSpacing = 1
+        minimumInteritemSpacing = 0
+    }
+
+    override func shouldInvalidateLayout(forBoundsChange newBounds: CGRect) -> Bool {
+        return true
+    }
+}
+
+// MARK: VLCActionSheetCell
+class VLCActionSheetCell: UICollectionViewCell {
+
+    static let identifier = "VLCActionSheetCell"
+
+    let icon: UIImageView = {
+        let icon = UIImageView()
+        icon.translatesAutoresizingMaskIntoConstraints = false
+        icon.contentMode = .scaleAspectFit
+        return icon
+    }()
+
+    let name: UILabel = {
+        let name = UILabel()
+        name.font = UIFont.systemFont(ofSize: 15)
+        name.translatesAutoresizingMaskIntoConstraints = false
+        return name
+    }()
+
+    let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.spacing = 15.0
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupViews()
+    }
+
+    private func setupViews() {
+        stackView.addArrangedSubview(icon)
+        stackView.addArrangedSubview(name)
+        addSubview(stackView)
+
+        var guide: LayoutAnchorContainer = self
+
+        if #available(iOS 11.0, *) {
+            guide = safeAreaLayoutGuide
+        }
+        NSLayoutConstraint.activate([
+            icon.heightAnchor.constraint(equalToConstant: 25),
```

Comment:
Maybe add a constant at the top of the file to easy future adaption of this constant?